src/main/java/uk/gov/hmcts/ccd/security/filters/LoggingFilter.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.ccd.security.filters;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.filter.OncePerRequestFilter;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+@Slf4j
+public class LoggingFilter extends OncePerRequestFilter {
```

Comment:
I think this name is a bit too generic. We can't log everything with this filter. Maybe a bit more specific name would help